core/workspace_svg.js

```diff
@@ -1777,7 +1777,6 @@ Blockly.WorkspaceSvg.prototype.cancelCurrentGesture = function() {
 /**
  * Get the audio manager for this workspace.
  * @return {Blockly.WorkspaceAudio} The audio manager for this workspace.
- * @package
```

Comment:
Blockly Games wants to play with Blockly's audio.  By removing @package, it can.  However, if you want to keep this private, just say so and I'll recreate the audio code in Blockly Games.